phoenix-core/src/main/java/org/apache/phoenix/util/SchemaUtil.java

```diff
@@ -540,6 +540,12 @@ public static boolean isSequenceTable(byte[] tableName) {
                 || Bytes.compareTo(tableName, SchemaUtil
                         .getPhysicalTableName(PhoenixDatabaseMetaData.SYSTEM_SEQUENCE_NAME_BYTES, true).getName()) == 0;
     }
+
+    public static boolean isTaskTable(byte[] tableName) {
```

Comment:
There are also some other places in the code where we do special things for SYSTEM tables, one of the examples is that we disable stats for SYSTEM tables. It would good if you can make that pass to make things consistent across code.

Reply:
Checkout `StatisticsUtil` class. This is where we disable it.

Reply:
Will do